src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -0,0 +1,41 @@
+package roomescape.controller;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.Reservation;
+import roomescape.Reservations;
+
+@RestController
+@RequestMapping("/reservations")
+public class ReservationController {
+
+    private final Reservations reservations = new Reservations();
+    private final AtomicLong index = new AtomicLong(1);
+
+    @GetMapping()
```

Comment:
```suggestion
    @GetMapping
```

채울 값이 없다면 `()` 생략 가능해요~

Reply:
앗 그렇네요. 감사합니다 :)